### ISSUE TITLE
Past Events are fetched on /events page

### ISSUE THREAD

**Issue by insertmike** [enucs/enucs#29]
**Problem Context:** All events are displayed under 'Upcoming Events' heading at https://enucs.org.uk/events/  
**Suggestion:** Either display upcoming events only or divide the section to two sub-sections - 'Upcoming Events' and 'Past Events'.

**Comment by glugg23** [comment]
Hi Mike, I've got this fixed on the feature/events-by-year branch, it's just not been merged into develop or put live, since I was waiting on some CSS from @MrRa1n

**Comment by insertmike** [comment]
Issue fixed on feature/events-by-year branch from @glugg23.
